packages/pass-style/src/types.d.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-use-before-define */
+import type { Checker as _Checker } from '@endo/common/ident-checker.js';
```

Comment:
Is this, together with line 208 below `export type Checker = _Checker;`, really the best way in type script to import-and-reexport a type from elsewhere?

Reply:
I defer to @turadg. I don’t recall whether this was a specific limitation for ambient types.

Reply:
There's a simpler way. I just pushed it.

Reply:
Thanks @turadg , resolving